src/displayapp/screens/WeatherSymbols.cpp

```diff
@@ -34,3 +34,28 @@ const char* Pinetime::Applications::Screens::Symbols::GetSymbol(const Pinetime::
       break;
   }
 }
+
+const char* Pinetime::Applications::Screens::Symbols::GetSimpleCondition(const Pinetime::Controllers::SimpleWeatherService::Icons icon) {
+  switch (icon) {
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Sun:
+      return "Clear";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::CloudsSun:
+      return "Cloudy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Clouds:
+      return "Cloudy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::BrokenClouds:
+      return "Cloudy";
```

Comment:
I see what you did here. Clever.
It will conflict with the weather app PR, so we'll need to find a good middle ground if both make it into `main`

Reply:
Okay, I see this is a different, simplified, function. Makes sense now! 👍